extensions/blocks/pinterest/pinterest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Pinterest Block.
+ *
+ * @since 7.x
```

Comment:
I think we can aim to ship this in `8.0.0`.

Reply:
👍🏻 f85dff4c2